reg_tests/CTestList.cmake

```diff
@@ -147,10 +145,12 @@ add_test_r_np(periodic3dEdge 8)
 add_test_r(quad9HC 2)
 add_test_r(steadyTaylorVortex 4 6)
 add_test_r_rst2(steadyTaylorVortex_P4 8)
-add_test_r_rst2(hoVortex_P2 8)
```

Comment:
This test is being removed as part of this pull request. Is this intentional?

Reply:
Relocated.